src/wp-includes/class-wp-theme-json.php

```diff
@@ -3527,6 +3593,13 @@ public static function get_from_editor_settings( $settings ) {
 			$theme_settings['settings']['spacing']['padding'] = $settings['enableCustomSpacing'];
 		}
 
+		if ( isset( $settings['spacingSizes'] ) ) {
+			if ( ! isset( $theme_settings['settings']['spacing'] ) ) {
+				$theme_settings['settings']['spacing'] = array();
+			}
+			$theme_settings['settings']['spacing']['spacingSizes'] = $settings['spacingSizes'];
```

Comment:
As with [above](https://github.com/WordPress/wordpress-develop/pull/6616/files#diff-7bab66e2c26e312df49f13ff601b4ab55601e876c21abdcf21cd884def623226R157), you might get away with setting the value on the array you wish to create, e.g.,  only `$theme_settings['settings']['spacing']['spacingSizes'] = $settings['spacingSizes'];`